### PR TITLE
Fixed NixOs vendoring issue

### PR DESCRIPTION
**Pull request by akorchyn**
Resolves: #392

You can test it by
```
cargo package
```

### CODE REVIEW

**Comment by akorchyn** [comment]
Sorry, I missed one spot in the build.rs :)
Should be fine now, I believe.

File: Cargo.toml [hunk 1]
```diff
@@ -27,17 +32,17 @@ blst = { version = "0.3.11", default-features = false }
 hex = { version = "0.4.2", default-features = false, features = ["alloc"] }
 libc = { version = "0.2", default-features = false }
 serde = { version = "1.0", optional = true, default-features = false, features = [
-    "alloc",
-    "derive",
+	"alloc",
+	"derive",
```
Comment (ralexstokes) [inline Cargo.toml:36]: not a big deal, but do we need to change the whitespace here?

File: Cargo.toml [hunk 2]
```diff
@@ -27,17 +32,17 @@ blst = { version = "0.3.11", default-features = false }
...
 ] }
 
 [dev-dependencies]
 criterion = "0.5.1"
 glob = "0.3.1"
 rand = "0.8.5"
 serde = { version = "1.0", features = ["derive"] }
-serde_yaml = "0.9.17"
 serde_json = "1.0.105"
+serde_yaml = "0.9.17"
```
Comment (ralexstokes) [inline Cargo.toml:45]: same question w/ reordering the deps here

**Review by ralexstokes — COMMENTED** [review]
nice! thanks for this :)

haven't tested out e.g. `cargo package` or `cargo vendor` yet but the diff generally looks good

File: Cargo.toml (hunk 2 above)
Comment (jtraglia) [reply Cargo.toml:45]: I think this is a good change. It was alphabetically out-of-order before.

File: Cargo.toml (hunk 1 above)
Comment (akorchyn) [reply Cargo.toml]: Restored original whitespaces.

**Review by jtraglia — APPROVED** [review]
LGTM 🥇thanks again @akorchyn! Just want some confirmations from the team before merging.

* @ralexstokes could you confirm that this fixes your issue?
* @Rjected & @pawanjay176 will this still work with Reth & Lighthouse?

**Review by pawanjay176 — APPROVED** [review]
Ran CI on lighthouse with this branch and it looks fine, changes also make sense to me

**Review by Rjected — APPROVED** [review]
this looks good to me, should work with revm / reth / alloy. moving cargo.toml to the top level makes sense

**Pull request merged**